### DISCUSSION TITLE
I'm having trouble entering decimal numbers in FluentNumberField when using EditForm.(Fluent UI Blazor)

### DISCUSSION THREAD

**Discussion by tomokusaba** [microsoft/fast#6833] [Q&A]
# https://github.com/microsoft/fluentui-blazor/discussions/733

> I looked at this some more and the error is originating from the underlying web component (fluent-number-field). You can easily verify that in your source by replacing the first FluentNumberField with a fluent-number field. The problem will need to be solved there first, I'm afraid. Please create an issue in the FAST repo.

I'm running into trouble like the one in the title in Fluent UI Blazor
So, in Fluent UI Blazor's Discussions, we had the following exchange:

At this point, I wondered if there might be a problem with the underlying fluent-number.

**Comment by LuohuaRain** [comment]
Maybe is it https://github.com/microsoft/fast/issues/6617 relevant?

**Reply by tomokusaba** [reply]
may be..
It is true
